addons/website/models/mixins.py

```diff
@@ -50,7 +50,7 @@ def _default_website_meta(self):
             'og:type': 'website',
             'og:title': title,
             'og:site_name': company.name,
-            'og:url': request.httprequest.url,
+            'og:url': request.website._get_canonical_url(canonical_params=False),
```

Comment:
As `_get_canonical_url()` purpose is not (only) to get the translated version of the URL but much more than just that, probably just calling `url_for()` is better.
```py
url_join(request.httprequest.url_root, url_for(request.httprequest.path))
```

Side note:
1. `environ['REQUEST_URI']` could have been used (it keeps the original path before rerouting with the lang) but is not supported in werkzeug 0.14 (only in 0.15 and +).
2. `request.rerouting` also holds the original path, but the code would look a bit ugly, need to check if exists etc

Courtesy of @JKE-be

Reply:
@rdeodoo Great! I changed the code this way. Could you check it again please?